docs/src/pages/guides/guide.web.mdx

```diff
@@ -0,0 +1,15 @@
+## Guides
+
+Amplify guides are meant to give you a more in-depth understanding of how to use the Amplify UI components and theming to build out common functionality, end-to-end solutions, and frequently asked for workflows.
+
+## Overview
+
+Guides are organized by topic, but some guides may overlap multiple topics (i.e. Authentication & Theming) to demonstrate how to integrate multiple services together.
+
+Guides may combine the CLI and libraries together in a single tutorial. This is done to teach an end-to-end solution using both the service creation and configuration as well as connecting to and interacting with the service from a client-side application.
+
+## Contributing
+
+If there is a guide or idea that you would like to see implemented that is not listed, open a GitHub Issue with the details of the use case and it will be reviewed by our team and considered.
```

Comment:
```suggestion
If there is a guide or idea that you would like to see implemented that is not listed, open a [GitHub Issue](https://github.com/aws-amplify/amplify-ui/issues/new/choose) with the details of the use case and it will be reviewed by our team and considered.
```

Reply:
👍

Reply:
7c6ee573f1e15b8407892dfd8552ea416e8cdf63